src/libraries/Microsoft.PowerFx.Core/App/IExternalEnabledFeatures.cs

```diff
@@ -18,10 +18,6 @@ internal interface IExternalEnabledFeatures
         bool IsUseDisplayNameMetadataEnabled { get; }
 
         bool IsDynamicSchemaEnabled { get; }
-
-        bool IsEnhancedComponentFunctionPropertyEnabled { get; }
-
-        bool IsComponentFunctionPropertyDataflowEnabled { get; }
```

Comment:
Since this may affect Power Apps, please only complete this PR once you have the changes ready in the PA repository.

Reply:
correct, I don't intend to check this one in until the PAClient PR is ready as well